maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .book-info-container {
+    flex-grow: 0;
+    margin-right: 0;
+  }
+
+  .book-title-container {
+    margin-bottom: 0;
+  }
+
+  .book-title {
+    font-size: 20px;
+    margin-right: 10px;
+  }
+
+  .book-gender {
+    font-size: 16px;
+  }
+
+  .book-detail-name {
+    font-size: 14px;
+    margin-right: 10px;
+    margin-bottom: 20px;
+  }
+
+  .book-detail-value {
+    font-size: 14px;
+    margin-bottom: 0;
+  }
+
+  .book-container {
+    align-items: center;
+    box-shadow: 4px 4px 10px 0 $black-05;
+    flex-direction: column;
+    max-width: 330px;
+    margin: 30px auto;
+    padding: 30px;
+    position: relative;
+    width: 100%;
```

Comment:
These two `position: relative` and `width: 100%;` are already set outside the media query. No need to set them again.